### DISCUSSION TITLE
Moving Plots

### DISCUSSION THREAD

**Discussion by freebst** [Chia-Network/chia-blockchain#2411] [Q&A] [answered]
I have a few machines that are creating plots.  After the plots are created I've moved them to a central machine to be farmed.  Is there any benefit to leaving the plots on the original machine to also be farmed?

I've also seen and read these two pages, but don't think I am ready to dabble in an overly complicated setup atm.
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Architecture
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

Is it okay to use the same 24 word mnemonic on several different computers?  

Will multi plotter, single farming setups be more easily set-up in the GUI or am I missing something obvious?

**Comment by cccat6** [comment, marked as answer]
No difference between leaving it in the original machine or move it to another machine.
24 words is your key to the wallet and everything. You can use it on many machines.
Multiple plotters and a single farmer is a general setup of Chia mining.
Suggesting use CLI for plotting based for many benefits.

**Reply by freebst** [reply]
Thanks for replying!  So just to verify, is there any benefit to having the exact same plots being farmed on different machines?  I'm guessing no because they're all under the same key/wallet.  Is that correct?

**Reply by cccat6** [reply]
Otherwise you are no need to plotting... Just copy paste an existing plot...
